src/us/kbase/workspace/test/database/mongo/ObjectListerTest.java

```diff
@@ -144,7 +148,8 @@ private DBObject getProjection() {
 	
 	private DBObject getProjection(final boolean includeMeta) {
 		final BasicDBObject p = new BasicDBObject();
-		Stream.of("ver", "type", "savedate", "savedby", "chksum", "size", "id", "ws")
+		Stream.of("ver", "tyname", "tymaj", "tymin", "savedate", "savedby", "chksum", "size",
+				"id", "ws")
```

Comment:
Er, what I meant to say was create a set of variables representing those strings. Variable-ise?

Reply:
For actual code I would agree but for tests I'm less concerned about it since the tests should fail if there's a typo, and making the same typo in many places (and thus allowing for the possibility that all tests could pass when they shouldn't) seems improbable. 

In general I tend to shy away from too much variableization and DRYing in tests because there's a tendency to wind up recapitulating  the actual code in the test code. Hence I try to use literals and construct data structures per test and avoid helper methods that, again, start recapitulating the code under test.